tests/integration/ganache/test_search_and_filter_assets.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import time
+
+from ocean_lib.structures.file_objects import UrlFile
+
+
+def test_search_and_filter_assets_by_tag(
```

Comment:
Why have this function at all if we are testing the readme? 

We cab definitely remove it if we remove the new method.

In general it's also a big DRY violation. We have to be working to reduce DRY violations, not the other way around.

Reply:
I got used when we created separate tests for README functionality, I'll make sure to delete it.